pyproject.toml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tool.bandit]
+exclude_dirs = ["/venv/"]
+[tool.bandit.assert_used]
+skips = ["*/*test.py", "*/test_*.py"]
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+# TODO: Enable this after migrate unittest to pytest
```

Comment:
Why? Aren't we running tests right now?

Reply:
The current test coverage is 39%.
There is a story to rewrite the unit test with pytest and increase the coverage.

Currently also working a story for integration test.